index.html

```diff
@@ -57,41 +64,43 @@ <h3>CANOPY</h3>
           <li>css</li>
           <li>javascript</li>
         </ul>
-        <button type='button'>See Project</button>
+        <button type='button' class="btn">See Project</button>
       </section>
     
       <section class="card">
-        <img src="images/second.png" alt="" aria-hidden="true">
-        <h2> Multi-Post <br> Stories </h2>
+        <img src="images/desk2.png" alt="" aria-hidden="true" class="bg">
```

Comment:
Your image is not scaling properly in the mobile version. This makes your text go out of the border. Kindly, fix it
![image](https://user-images.githubusercontent.com/12239821/167153670-c2ced7b6-02f5-4b72-90cc-d6a14852e728.png)